### ISSUE TITLE
Error detecting `Vivo` devices

### ISSUE THREAD

**Issue by ayumi-cloud** [WhichBrowser/Parser-PHP#253]
UA:

```
Mozilla/5.0 (Linux; U; Android 9; th; vivo 1915) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.137 Mobile Tenta/3.1.6 Build/2162 Safari/537.36
```

Result:

```
Calculated:
browser:
    name: 'Tenta Browser'
    family: { name: Chrome, version: 64 }
    version: 3.1.6
    type: browser
engine:
    name: Blink
os:
    name: Android
    version: '9'
device:
    type: mobile
    subtype: smart
    model: 'vivo 1915) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.137 Mobile Tenta/3.1.6'
```

The `device->model` is displaying junk result.

Expected results:

```
device:
    model: Y19
```

**Comment by ayumi-cloud** [comment]
I leave the admins to fix this issue.

**Comment by NielsLeenheer** [comment]
Fixed in upcoming 2.1.1
